content/python/concepts/sets/terms/difference/difference.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: '.difference()'
+Description: 'Returns a new set of objects unique between two or more sets.'
```

Comment:
The summary of the difference function, used repeatedly, is misleading. It implies that we're returning items unique among all sets, when were actually only returning items unique in the first set. The descriptions should be rewritten to make this clear.

Reply:
You're right. This needs to be reworded. What do we think of this possible edit:

"`Description`: Returns a new set of objects unique to a given set when compared to other sets."

Reply:
That looks good 😃